3_extracting_features/3.compile_DP_projects.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+jupyter nbconvert --to python compile_DP_projects.ipynb
+python compile-DP-projects.py
```

Comment:
```suggestion
python compile-DP-projects.py

```

Here too

Reply:
Same as comment above, I have no idea why I don't see that red! :cry: